### PR TITLE
1st version of a responsive design - mobile first

### PR DESCRIPTION
**Pull request by TeigtraeumeSchachner**
I adjusted my 1st project into a responsive design with the focus on smartphone sizes. 

The workbook template helped me a lot with the definition of the general layouts according to the breakpoints. Unfortunately, I did not have that much time to implement it accordingly. I got stuck on the layout of the menu on the welcome page (index.html). I want the basic layout to consist of pictures with a hover effect changing to text - more precisely changing to a link to the corresponding page. That didn't work out so far - I will need to work on this more in detail.

### CODE REVIEW

**Comment by HIPPIEKICK** [comment]
Hi Melanie! I think you forgot to include your image folder in your project. Your page looks like this:
Skärmavbild 2020-10-28 kl  11 02 57

If you want me to do a review anyway, please just respond to this comment (or upload the images :) )

**Comment by TeigtraeumeSchachner** [comment]
Hi Matilda,
Thanks for the hint! I had not even realized that the img-folder was not there.
You should see them now, hopefully.

**Comment by HIPPIEKICK** [comment]
Awesome, thanks!

File: code/index.html [hunk 1]
```diff
@@ -1,18 +1,49 @@
...
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body>
+    <h1>Hallo liebe(r) Freund(in) der Brotkultur!</h1>
+    <p>Schön, dass du den virtuellen Weg zu meinem Brotprojekt gefunden hast. :-) 
+    </p>
+    <p>Ich möchte dir auf den folgenden Seiten einen Überblick dazu geben.
+        Und natürlich würde ich mich freuen, dich bald einmal persönlich kennenzulernen.
+    </p>
```
Comment (HIPPIEKICK) [inline code/index.html:18]: Would be cool if these elements were wrapped in some other element, like a `header`. Then we could style that header to look something like this:
Skärmavbild 2020-10-28 kl  11 48 58

This is of course just an example, but maybe something to play around with? 😇 
**HTML:**
```
<header>
  <h1>Hallo liebe(r) Freund(in) der Brotkultur!</h1>
  <div class="subtitle">
    <p>Schön, dass du den virtuellen Weg zu meinem Brotprojekt gefunden hast. :-) 
    </p>
    <p>Ich möchte dir auf den folgenden Seiten einen Überblick dazu geben.
      Und natürlich würde ich mich freuen, dich bald einmal persönlich kennenzulernen.
    </p>
  </div>
</header>
```
**CSS:**
```
header {
  background: goldenrod;
}

header h1, header p {
  padding: 20px;
  margin: 0;
}

.subtitle {
  background: gold;
  width: 100%;
}
```

File: code/index.html [hunk 2]
```diff
@@ -1,18 +1,49 @@
...
+
+    <section class="menu-container">
+        <a href="./">Willkommen!</a>
+        <div class="menu-line">
+            <img src="/img/Logo.jpg" alt="Wer steckt hinter Teigträume Schachner?" class="menu-img">
+            <div class="overlay">
+            <a class="menu-text" href="./about.html">Wer steckt hinter Teigträume Schachner?</a>
+            </div>
```
Comment (HIPPIEKICK) [inline code/index.html:26]: Here's an example of how you can achieve overlay: https://codepen.io/hippiekick/pen/qBNpwqw?editors=1000

File: code/philosophy.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        </div>
+        <div class="flex-item">
+            <img src="" alt="">
+            <p><mark>Nüsse</mark> und <mark>Trockenfrüchte</mark> beziehe ich hauptsächlich bei der <a href="http://www.sun-snack.ch/"> Sun-Snack AG</a> in St. Margrethen.</p>
+        </div>
+        <div class="flex-item">
+            <img src="" alt="">
+            <p><mark>Weitere Zutaten</mark>, wie z.B. Zucker oder Salz hole ich bei der <a href="https://www.lindmuehle.ch/de/Privatverkauf-Region-Ostschweiz.php"> Erlenmühle Gossau</a>.</p>
+        </div>
+    </section>
```
Comment (HIPPIEKICK) [inline code/philosophy.html:42]: Yey, flexbox! Would be super nice if you used in on the whole page to center your content 🤩

Comment (HIPPIEKICK) [reply code/philosophy.html:42]: If you wrapped all element inside a `<main>` element and add this to the `<body>` and `main` elements:
```
body {
  display: flex;
  flex-direction: column;
  align-items: center;
}

main {
  margin-top: 50px;
    max-width: 70%;
}
```
you would get a little more space around like this:

Skärmavbild 2020-10-28 kl  14 10 49

**Review by HIPPIEKICK — COMMENTED** [review]
Good job trying out responsive web design! I tried to answer your questions in the comments and added some examples. Keep up the good work and continue exploring all the awesome things you can do with flexbox. Here are two games you could play to practice, they are actually quite fun 🤩

- [Flexbox Froggy](https://flexboxfroggy.com/)
- [Flexbox Zombies](https://flexboxzombies.com/p/flexbox-zombies)

**Pull request closed without merging**